### ISSUE TITLE
Casino off egg; broken popup

### ISSUE THREAD

**Issue by StoneCypher** [pmotschmann/Evolve#334]
The casino off button was an egg (sorry, neglected to get a screen) and when I clicked it, I got this popup, which appears to be permanent:

![image](https://user-images.githubusercontent.com/77482/79080685-ea570c00-7ccb-11ea-982b-30955068d298.png)

**Comment by StoneCypher** [comment]
oh hey, i have two more of the egg

![image](https://user-images.githubusercontent.com/77482/79080717-17a3ba00-7ccc-11ea-944d-33f08fcb65d4.png)

![image](https://user-images.githubusercontent.com/77482/79080724-2be7b700-7ccc-11ea-9af6-ea0a426db5cc.png)

**Comment by StoneCypher** [comment]
huh, the second one i clicked and it gave me plasmids

third one disappeared before i knew i was supposed to click them

maybe the popup is unrelated

**Comment by StoneCypher** [comment]
... oh.  it's easter.  they're easter eggs.

lol this is what i get for not being religious

**Comment by StoneCypher** [comment]
The dialog's contents have changed.  However, it stays open, even when I switch away from meaningful screens.  Unless capped by left content, it follows the bottom edge of the content panel, despite not being in the content panel, but rather the left panel

![image](https://user-images.githubusercontent.com/77482/79080847-4706f680-7ccd-11ea-9ba7-451c8124c032.png)

to

![image](https://user-images.githubusercontent.com/77482/79080864-5c7c2080-7ccd-11ea-805b-535a5d8b7aef.png)

Using this, I can get the same popup twice.  It wasn't until I did so that I noticed something weird

![image](https://user-images.githubusercontent.com/77482/79080878-79b0ef00-7ccd-11ea-8213-08e3ff725426.png)

If you look very carefully, the text in the two copies of the popup is rendering differently.  The misplaced one is sharp; the correctly placed one is slightly fuzzy.  This suggests it might be being scaled  I think it's the `transform: translate3d` being used to position it, but I can't find a way to inspect it that doesn't drop the hover, so I'm not certain
